src/main/scala/mesosphere/marathon/api/v2/EventSubscriptionsResource.scala

```diff
@@ -35,7 +36,9 @@ class EventSubscriptionsResource @Inject() (val config: MarathonConf) extends Re
   @Timed
   def subscribe(@Context req: HttpServletRequest, @QueryParam("callbackUrl") callbackUrl: String): Response = {
     validateSubscriptionService()
-    val future: Future[MarathonEvent] = service.handleSubscriptionEvent(Subscribe(req.getRemoteAddr, callbackUrl))
+    validateUrl(callbackUrl)
```

Comment:
I moved validation function to `src/main/scala/mesosphere/marathon/api/v2/Validation.scala` and reused it in `src/main/scala/mesosphere/marathon/event/http/HttpEventModule.scala`.